USERGUIDE.md

```diff
@@ -1074,6 +1074,25 @@ window.addEventListener(MiniAppKeyboardEvents.KEYBOARDHIDDEN, function (e) {
 }
 ```
 
+<a id="miniapp-close-event-listner></a>
```

Comment:
```suggestion
<a id="miniapp-close></a>
```

Reply:
Thank you I have updated in the latest commit.